ui.py

```diff
@@ -28,9 +28,14 @@ def show_books(books):
 
     if books:
         for book in books:
-            print(book)
+            if book == books[0]:
```

Comment:
Are these codes extra that you added to this branch?

Reply:
Hi, yes,

There is a few lines more than 31, my changes expand this code to line 38.

Reply:
blankline is the branch, this code added the blanklines. Sorry if I'm misunderstanding!! Still getting used to working in git like this.

Reply:
I do wonder if it would be better to put them on a different branch instead? What do you think? Not the "\n" part, but the: if book == books[0]: & elif book == books[(len(books) - 1)]: I don't know if that's how Clara would want, but it was just a thought.

Reply:
That's necessary for it to not print a blank line between each book in the list. The if/elif/else function makes sure a blank line is put before the first book, after the last book, and nothing in between.

Reply:
Ok, I see what you're saying. Are you running into any errors with this code? I ran it on my end with an error. TypeError: can only concatenate str (not "Book") to str

Reply:
Shoot, yes I'm seeing that too with the search option. I'll keep working on it. Hold on this one. Thank you!

Reply:
Maybe the code can be a bit shorter? What do you think? I remember Clara mentioning that the codes are a line or two.